.github/workflows/ci.yml

```diff
@@ -25,16 +25,16 @@ jobs:
       image: ${{ matrix.docker_image }}
     steps:
     - uses: actions/checkout@v2
-    - uses: ros-tooling/setup-ros@0.1.2
+    - uses: ros-tooling/setup-ros@0.2.0
```

Comment:
@pablogs9 maybe you also have to change the two `branches` values to `galactic` as in this pull request: https://github.com/ros2/rclc/pull/91

Reply:
They are already changed I guess: https://github.com/ros2/rclc/blob/6b755f44a4c6c8a83104c142b3fc0eb9651b65e3/.github/workflows/ci.yml#L5